cmd/root.go

```diff
@@ -64,6 +69,10 @@ func init() {
 
 	// VaultStorer config
 	flag.StringVar(&krokArgs.fileVault.Location, "krok-file-vault-location", "/tmp/krok/vault", "--krok-file-vault-location /tmp/krok/vault")
+
+	// OAuth
+	flag.StringVar(&krokArgs.server.GoogleClientID, "google-client-id", "", "--google-client-id my-client-id}")
```

Comment:
I wonder if we should error out if these are not set? The application is effectively unusable in that case?